tools/ClassGen/NodeGen.cpp

```diff
@@ -1542,6 +1542,27 @@ int main(int argc, char **argv) {
                     "V4 by outputting indices and scalar tensor with number of "
                     "valid indices. It pads the rest with global MIN box.");
 
+  BB.newNode("TFLiteDetectionPostProcess")
+      .addInput("Boxes")
+      .addInput("Scores")
+      .addInput("Anchors")
+      .addMember(MemberType::Unsigned, "NumClasses")
+      .addMember(MemberType::Unsigned, "MaxDetections")
+      .addMember(MemberType::Unsigned, "MaxClassesPerDetection")
+      .addMember(MemberType::Unsigned, "MaxDetectionsPerClass")
+      .addMember(MemberType::Float, "IouThreshold")
+      .addMember(MemberType::Float, "ScoreThreshold")
+      .addMember(MemberType::Float, "XScale")
+      .addMember(MemberType::Float, "YScale")
+      .addMember(MemberType::Float, "HScale")
+      .addMember(MemberType::Float, "WScale")
+      .addMember(MemberType::Boolean, "RegularNMS")
+      .addResultFromCtorArg("DetectionBoxes")
+      .addResultFromCtorArg("DetectionClasses")
+      .addResultFromCtorArg("DetectionScores")
+      .addResultFromCtorArg("NumDetections")
+      .setDocstring("This is a TensorFlowLite version of NonMaxSuppresion.");
```

Comment:
Could you please add more info about this op? Like, how each params work? Thanks!

Reply:
I added more info about this op.
Done!